lucas_scheinkerman/ROS_investigation.md

```diff
@@ -0,0 +1,222 @@
+/create1/move_base/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
+/create1/move_base/goal [move_base_msgs/MoveBaseActionGoal]
+/create1/amcl_pose [geometry_msgs/PoseWithCovarianceStamped]
+/create1/bumper [ca_msgs/Bumper]
+/gazebo/link_states [gazebo_msgs/LinkStates]
+/create1/raspicam/parameter_updates [dynamic_reconfigure/Config]
+/create1/move_base/global_costmap/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
+/cmd_vel [geometry_msgs/Twist]
+/gazebo/model_states [gazebo_msgs/ModelStates]
+/clock [rosgraph_msgs/Clock]
+/create1/move_base/NavfnROS/plan [nav_msgs/Path]
+/create1/raspicam/image_raw [sensor_msgs/Image]
+/create1/move_base/global_costmap/obstacle_layer/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
+/create1/laser/scan [sensor_msgs/LaserScan]
+/create1/move_base/status [actionlib_msgs/GoalStatusArray]
+/create1/move_base/global_costmap/static_layer/parameter_descriptions [dynamic_reconfigure/ConfigDescription]
+/create1/move_base/DWAPlannerROS/local_plan [nav_msgs/Path]
+/gazebo/parameter_updates [dynamic_reconfigure/Config]
+
+
+Subscribed topics:
+ /create1/move_base/cancel [actionlib_msgs/GoalID]
+ /create1/move_base/local_costmap/footprint [geometry_msgs/PolygonStamped]
+ /gazebo/set_link_state [gazebo_msgs/LinkState]
+ /tf [tf2_msgs/TFMessage] 3 subscribers
+ /create1/move_base_simple/goal [geometry_msgs/PoseStamped]
+ /create1/mobile_base/sensors/bumper_pointcloud [sensor_msgs/PointCloud2]
+ /create1/joint_states [sensor_msgs/JointState]
+ /create1/cmd_vel [geometry_msgs/Twist]
+ /create1/move_base/local_costmap/costmap_updates [map_msgs/OccupancyGridUpdate]
+ /create2/move_base/local_costmap/costmap_updates [map_msgs/OccupancyGridUpdate]
+ /tf_static [tf2_msgs/TFMessage] 3 subscribers
+ /create1/move_base/global_costmap/footprint [geometry_msgs/PolygonStamped]
+ /map_updates [map_msgs/OccupancyGridUpdate]
+ /map [nav_msgs/OccupancyGrid] 3 subscribers
+ /rosout [rosgraph_msgs/Log]
+ /create1/gts [nav_msgs/Odometry]
+ /create1/initialpose [geometry_msgs/PoseWithCovarianceStamped]
+ /create1/odom [nav_msgs/Odometry]
+ /create1/nodelet_manager/bond [bond/Status] 2 subscribers
+ /gazebo/set_model_state [gazebo_msgs/ModelState]
+ /create1/move_base/goal [move_base_msgs/MoveBaseActionGoal]
+ /create1/cliff [ca_msgs/Cliff]
+ /create2/move_base/local_costmap/costmap [nav_msgs/OccupancyGrid]
+ /clock [rosgraph_msgs/Clock] 12 subscribers
+ /create1/move_base/local_costmap/costmap [nav_msgs/OccupancyGrid]
+ /create1/bumper [ca_msgs/Bumper]
+ /create1/laser/scan [sensor_msgs/LaserScan]
+```
+
+## Listening to a specific topic
+
+This assignment was completed by executing the command 'rostopic echo /create1/cmd_vel', which corresponds to the topic that has information about the angular and linear velocity of the simulated robot. Simultaneously with the printing made by this command, the robot was being moved with the 'keyboard_teleop' tool. The result was:
+
+```console
```

Comment:
Please, decrease the amount of information here. It's not necessary to print them all.